pkg/cmd/policy_enable.go

```diff
@@ -48,15 +53,69 @@ func newPolicyEnableCmd() *cobra.Command {
 				version = &cliArgs[1]
 			}
 
+			// Load the configuration from the user-specified JSON file into config object.
+			var config map[string]*json.RawMessage
+			if args.config != "" {
+				// config, err = loadJSONConfigFile(args.configFile)
+				config, err = loadPolicyConfigFromFile(args.config)
+				if err != nil {
+					return err
+				}
+			}
+
 			// Attempt to enable the Policy Pack.
-			return policyPack.Enable(commandContext(), args.policyGroup, backend.PolicyPackOperation{
-				VersionTag: version, Scopes: cancellationScopes})
+			return policyPack.Enable(commandContext(), args.policyGroup,
+				backend.PolicyPackOperation{
+					VersionTag: version,
+					Scopes:     cancellationScopes,
+					Config:     config,
+				})
 		}),
 	}
 
 	cmd.PersistentFlags().StringVar(
 		&args.policyGroup, "policy-group", "",
 		"The Policy Group for which the Policy Pack will be enabled; if not specified, the default Policy Group is used")
 
+	cmd.PersistentFlags().StringVar(
+		&args.config, "config", "",
+		"The file path for the policy pack configuration file")
+
 	return cmd
 }
+
+func loadPolicyConfigFromFile(file string) (map[string]*json.RawMessage, error) {
+	analyzerPolicyConfigMap, err := resourceanalyzer.LoadPolicyPackConfigFromFile(file)
+	if err != nil {
+		return nil, err
+	}
+
+	// Convert type map[string]plugin.AnalyzerPolicyConfig to map[string]*json.RawMessage.
+	config := map[string]*json.RawMessage{}
```

Comment:
Hmm not sure how this is working, but don't you normally have to use `make` for a map? 🤔

Reply:
It is equivalent to initalizing an empty map using make. Only difference is not including the curly braces at the end when using the make function.  `make(map[string]*json.RawMessage)`.

Reply:
Using `make` seems to be the more consistent pattern for us from a quick glance in our code base. So went and just changed to using make while I was in there.  😃